pdf_generator.py

```diff
@@ -0,0 +1,171 @@
+import fpdf
+import copy
+class PdfGenerator:
```

Comment:
The name PdfGenerator is a bit misleading, this class does create an activity report in Pdf format.
What would you say about `ActivityReportRenderer`